src/version.h.in

```diff
@@ -0,0 +1,11 @@
+// Get version number from CMAKE
+#ifndef INCLUDE_GUARD
+#define INCLUDE_GUARD
+
+#define PROJECT_NAME "@PROJECT_NAME@"
+#define PROJECT_VER  "@PROJECT_VERSION@"
+#define PROJECT_VER_MAJOR "@PROJECT_VERSION_MAJOR@"
```

Comment:
If we're using the `VERSION "string"` CMake function, do we need the `_VER_MAJOR`/`MINOR`/`PATCH`? I vote we don't.

Reply:
Agree!